cmd/podman/common.go

```diff
@@ -313,6 +313,10 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"hostname", "h", "",
 		"Set container hostname",
 	)
+	createFlags.Bool(
+		"http-proxy", true,
+		"Set proxy environment variables in container based on the host proxy vars",
```

Comment:
nit of a nit, and I would'nt hold up the PR for this, but s/container/the container/

Reply:
fixed in #3047